code/__DEFINES/reagents.dm

```diff
@@ -68,7 +68,7 @@
 #define CHEMICAL_MAXIMUM_TEMPERATURE 99999
 
 ///The default purity of all non reacted reagents
-#define REAGENT_STANDARD_PURITY 0.75
+#define REAGENT_STANDARD_PURITY 1
```

Comment:
I don't think this is a good idea, what does this gain in this PR alone?

In the ideal system, where two productions of chemicals are different from one another, purity is that metric that sets them apart. I wouldn't want default chems to already be maxed out. This should probably be kept as is.

Reply:
This has been reverted to 0.75